packages/typescript-estree/tests/lib/persistentParse.test.ts

```diff
@@ -141,7 +141,7 @@ function baseTests(
     fs.mkdirSync(path.join(PROJECT_DIR, 'src', 'bat'));
     fs.mkdirSync(path.join(PROJECT_DIR, 'src', 'bat', 'baz'));
 
-    const bazSlashBar = path.join('bat', 'baz', 'bar') as 'bat/baz/bar';
+    const bazSlashBar = 'bat/baz/bar' as const;
```

Comment:
@bradzacher you can't do `path.join` here as is not going to work on windows machines -> this field is used as key of
CONTENTS